wicket/src/ui/panes/update.rs

```diff
@@ -1018,6 +1055,160 @@ impl UpdatePane {
+                    assert!(
+                        rot.is_none(),
+                        "update item contains multiple RoT entries"
+                    );
+                    if artifact_version == installed_version {
+                        rot = Some(ComponentForceUpdateSelectionState {
+                            version: artifact_version,
+                            toggled_on: state
+                                .force_update_state
+                                .force_update_rot,
+                            selected: false, // set below
+                        });
+                    }
+                }
+                UpdateComponent::Sp => {
+                    assert!(
+                        sp.is_none(),
+                        "update item contains multiple RoT entries"
+                    );
+                    if artifact_version == installed_version {
+                        sp = Some(ComponentForceUpdateSelectionState {
+                            version: artifact_version,
+                            toggled_on: state
+                                .force_update_state
+                                .force_update_sp,
+                            selected: false, // set below
+                        });
+                    }
+                }
+                UpdateComponent::Host => unreachable!(), // skipped above
+            }
+        }
+
+        // If we only have one force-updateable component, mark it as selected;
+        // otherwise, respect the option currently selected in `State`.
+        match (rot.as_mut(), sp.as_mut()) {
+            (Some(rot), None) => rot.selected = true,
+            (None, Some(sp)) => sp.selected = true,
+            (Some(rot), Some(sp)) => {
+                if state.force_update_state.selected_component()
+                    == UpdateComponent::Rot
+                {
+                    rot.selected = true;
+                } else {
+                    sp.selected = true;
+                }
+            }
+            (None, None) => (),
+        }
+
+        Self { rot, sp }
+    }
+}
+
+impl ForceUpdateSelectionState {
+    fn num_spans(&self) -> usize {
+        usize::from(self.rot.is_some()) + usize::from(self.sp.is_some())
+    }
+
+    fn next_component(&self, state: &mut State) {
+        // Only move to the next component if we're showing more than 1.
+        if self.num_spans() > 1 {
+            state.force_update_state.next_component();
+        }
+    }
+
+    fn prev_component(&self, state: &mut State) {
+        // Only move to the prev component if we're showing more than 1.
+        if self.num_spans() > 1 {
+            state.force_update_state.prev_component();
+        }
+    }
+
+    fn help_text(&self) -> Vec<(&'static str, &'static str)> {
+        match self.num_spans() {
+            0 => vec![],
+            1 => vec![("Toggle", "Space")],
+            _ => vec![("Toggle", "Space"), ("Up", "Up"), ("Down", "Down")],
```

Comment:
I think this needs to be `<Space>` etc for consistency.

Reply:
So it does! Thanks.

Reply:
:+1: fixed in c38e13e